events/message_reaction_added_EVT.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  name: 'Message Reaction Added MOD',
+  name: 'Message Reaction Added',
```

Comment:
Changing the `name` property will break existing events that users might be using. Should probably revert all the changes in the `events` folder